tasks/config/run.js

```diff
@@ -59,6 +74,34 @@ module.exports = function (grunt) {
       ]
     },
 
+    seleniumServer: {
```

Comment:
This could be more DRY but I'm following convention from the rest of the file.  Let me know and I can refactor.